### PR TITLE
Wish list moved from github wiki to docs

### PR DESCRIPTION
**Pull request by anki-code**
Related to p.7 in #3838 and to #3806

### CODE REVIEW

File: docs/todo.rst [hunk 1]
```diff
@@ -15,6 +15,21 @@ For now we use Python syntax by adding the ``language`` to ``.gitattributes``:
...
+* Gitsome-style rich git(hub) client
+* Timeout context manager--will kill a ``with`` block if it takes too long
+* Macros to run a command in a specific container
+* Tools to go between ``namedtuple`` and shell-style tables
+* Able to use ``find``-like and ``grep``-like calls as iterables
+* ``touch`` and ``rm`` like xoreutils
+* `Keep <https://github.com/OrkoHunter/keep>`_ (With PTK keyboard shortcut)
+* Display docstring of python class/method/function being typed into xonsh prompt, like ptpython
+* udisks notifications and easy access
+* Easily (transparently) load/save data
```
Comment (scopatz) [inline docs/todo.rst:31]: I am not sure what data this is in relation to.

File: docs/todo.rst [hunk 2]
```diff
@@ -15,6 +15,21 @@ For now we use Python syntax by adding the ``language`` to ``.gitattributes``:
...
 
+xontribs
+----------------------------------------
+This is simply a list of things we wish existed as a xontrib.
+
+* Gitsome-style rich git(hub) client
+* Timeout context manager--will kill a ``with`` block if it takes too long
+* Macros to run a command in a specific container
+* Tools to go between ``namedtuple`` and shell-style tables
+* Able to use ``find``-like and ``grep``-like calls as iterables
```
Comment (scopatz) [inline docs/todo.rst:26]: You can do this now, with certain subprocess operators

**Comment by anki-code** [comment]
These items are out of date! Thanks! Closed.

**Comment by scopatz** [comment]
I think you can just remove those line :wink:

**Pull request closed without merging**